frontend/src/component/layout/MainLayout/MainLayout.tsx

```diff
@@ -70,6 +70,12 @@ const NewMainLayoutContent = styled(Grid)(({ theme }) => ({
         paddingRight: theme.spacing(1),
         margin: 0,
     },
+    [theme.breakpoints.up('xl')]: {
```

Comment:
<img width="1905" alt="Screenshot 2024-05-27 at 13 15 35" src="https://github.com/Unleash/unleash/assets/1394682/42f762a0-5512-427e-86c6-a9fb9c2ee0c4">

On very large screens use fixed width in the center